examples/mysql-kopia-repository-server/README.md

```diff
@@ -0,0 +1,423 @@
+# MySQL
+
+[MySQL](https://MySQL.org) is one of the most popular database servers in the world. Notable users include Wikipedia, Facebook and Google.
+
+## Introduction
+
+This chart bootstraps a single node MySQL deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.20+
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.90.0 installed in your cluster, let's assume in Namespace `kanister`
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+
+## Installing the Chart
+
+To install the MySQL database using the `bitnami` chart with the release name `mysql-release`:
+
+```bash
+# Add bitnami in your local chart repository
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+
+# Update your local chart repository
+$ helm repo update
+
+# Install the MySQL database
+
+> **Note**: The custom image for MySQL is required in order to utilize the Kopia Repository Server based functions.
```

Comment:
```suggestion
> **Note**: The custom image for MySQL is required because the Kopia Repository Server based functions (used in Blueprint), require `Kopia` binary to be available on application pod.
```